src/View/Requirements_Backend.php

```diff
@@ -631,13 +631,21 @@ public function javascriptTemplate($file, $vars, $uniquenessID = null)
      * @param string $file The CSS file to load, relative to site root
      * @param string $media Comma-separated list of media types to use in the link tag
      *                      (e.g. 'screen,projector')
+     * @param array $options List of options. Available options include:
+     * - 'integrity' : SubResource Integrity hash
+     * - 'crossorigin' : Cross-origin policy for the resource
      */
-    public function css($file, $media = null)
+    public function css($file, $media = null, $options = [])
     {
         $file = ModuleResourceLoader::singleton()->resolvePath($file);
 
+        $integrity = isset($options['integrity']) ? $options['integrity'] : null;
+        $crossorigin = isset($options['crossorigin']) ? $options['crossorigin'] : null;
```

Comment:
We have PHP 7.1 as a minimum in this branch now, so you could do `$crossorigin = $options['crossorigin'] ?? null;` to shorten it a little